packages/docusaurus-plugin-content-docs/src/__tests__/lastUpdate.test.ts

```diff
@@ -40,7 +40,7 @@ describe('lastUpdate', () => {
     expect(consoleMock).toHaveBeenCalledTimes(1);
     expect(consoleMock).toHaveBeenCalledWith(
       new Error(
-        `Command failed with exit code 128: git log -1 --format=%ct, %an ${nonExistingFilePath}`,
+        `Command failed with exit code 128: git log -1 --format=%ct, %an ${nonExistingFilePath}\nfatal: ambiguous argument '${nonExistingFilePath}': unknown revision or path not in the working tree.\nUse '--' to separate paths from revisions, like this:\n'git <command> [<revision>...] -- [<file>...]'`,
```

Comment:
any reason for this change?

Reply:
To pass the tests, apparently something has changed in execa v5 :man_shrugging:

Reply:
Not sure what's the easiest way to do that with Jest, but I think just matching the error message beginning should be fine.

Done this in another place:

```
    expect(consoleSpy.mock.calls[0][0].message).toMatch(/^Command failed with exit code 128/);
```

Reply:
I did it a little differently, the test passed, but Windows CI still  failed.